tests/report_tests/help_output_tests.py

```diff
@@ -45,7 +45,62 @@ def test_no_missing_plugin_descriptions(self):
                          "The following plugins are currently disabled:",
                          self.cmd_output.stdout, re.S).group(1).splitlines()
         for ln in _out:
+            ln = ln.split()
             # Ignore newlines
             if not ln:
                 continue
             assert len(ln) > 1, f"Plugin '{ln[0]}' missing description"
+
+    def test_plugin_formatting(self):
+        _out = re.search("The following plugins are currently enabled:(.*?)"
+                         "The following plugins are currently disabled:(.*?)"
+                         "The following options are available "
+                         "for ALL plugins:(.*?)"
+                         "The following plugin options are available:(.*?)"
+                         "Profiles:",
+                         self.cmd_output.stdout, re.S)
+        enabled_plugins = _out.group(1).splitlines()
+        disabled_plugins = _out.group(2).splitlines()
+        options = _out.group(3).splitlines()
+        plugin_options = _out.group(4).splitlines()
+        for plug in enabled_plugins:
+            # Ignore empty lines
+            if not plug.strip():
+                continue
+            self.assertRegex(plug, r' ([\S ]){20} ([\S ])*')
+        for plug in disabled_plugins:
+            if not plug.strip():
+                continue
+            self.assertRegex(plug, r' ([\S ]){20} (inactive[ ]{6}) ([\S ])*')
+        for opt in options:
+            if not opt.strip():
+                continue
+            self.assertRegex(opt, r' ([\S ]){25} ([\d ]{15}) ([\S ])*')
+        for opt in plugin_options:
+            if not opt.strip():
+                continue
+            self.assertRegex(
+                opt,
+                r' ([\S ]){30} (on[ ]{13}|off[ ]{12}|[\d ]{15}) ([\S ])*'
```

Comment:
The `(on[ ]{13}|off[ ]{12}|[\d ]{15})` does not match `mysql` or other string values of plugopts - see failing match on `' mysql.dbuser                   mysql           username for database dump collection'`.

Since plugin options default values can be arbitrary, I would simplify this test to "any string with optional trailing spaces of length (??) chars altogether".